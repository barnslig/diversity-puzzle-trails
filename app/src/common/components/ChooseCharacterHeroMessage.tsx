import { ArrowDownward } from "@mui/icons-material";
import { FormattedMessage } from "react-intl";
import * as React from "react";

import HeroMessage from "./HeroMessage";

import config from "../../config";

type ChooseCharacterHeroMessageProps = {};

const ChooseCharacterHeroMessage = (props: ChooseCharacterHeroMessageProps) => {
  return (
    <HeroMessage
      title={
        <FormattedMessage
          defaultMessage="Charakter wählen"
          description="onboarding choose character headline"
        />
      }
      description={
        <FormattedMessage
          defaultMessage="Scanne eine Charakterkarte um am Spiel teilzunehmen. Dein gewählter Charakter gibt dir bei jeder Aktion im Spiel einen Bonus!"
          description="onboarding choose character paragraph"
        />
      }
      after={
        <ArrowDownward
          sx={{
            marginBottom: 3,
            marginTop: 3,
            transform: config.featureMessages ? "" : "translateX(84px)",
          }}
        />
      }
    />
  );
};

export default ChooseCharacterHeroMessage;
