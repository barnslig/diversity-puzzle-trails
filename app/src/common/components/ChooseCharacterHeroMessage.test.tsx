import * as React from "react";
import render from "../testing/render";

import ChooseCharacterHeroMessage from "./ChooseCharacterHeroMessage";

it("renders correctly", () => {
  const { container } = render(<ChooseCharacterHeroMessage />);
  expect(container.firstChild).toMatchSnapshot();
});
