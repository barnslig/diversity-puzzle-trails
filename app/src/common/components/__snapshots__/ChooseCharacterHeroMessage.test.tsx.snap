// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<main
  class="MuiBox-root css-elj48x"
>
  <div
    class="MuiContainer-root MuiContainer-maxWidthSm css-cuefkz-MuiContainer-root"
  >
    <h2
      class="MuiTypography-root MuiTypography-h5 MuiTypography-gutterBottom css-h93ljk-MuiTypography-root"
    >
      Charakter wählen
    </h2>
    <p
      class="MuiTypography-root MuiTypography-body2 css-r40f8v-MuiTypography-root"
    >
      Scanne eine Charakterkarte um am Spiel teilzunehmen. Dein gewählter Charakter gibt dir bei jeder Aktion im Spiel einen Bonus!
    </p>
    <svg
      aria-hidden="true"
      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-q2ql64-MuiSvgIcon-root"
      data-testid="ArrowDownwardIcon"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="m20 12-1.41-1.41L13 16.17V4h-2v12.17l-5.58-5.59L4 12l8 8 8-8z"
      />
    </svg>
  </div>
</main>
`;
